pygmt/tests/test_grdsample.py

```diff
@@ -0,0 +1,44 @@
+"""
+Tests for grdsample.
+"""
+import os
+
+import pytest
+from pygmt import grdinfo, grdsample
+from pygmt.datasets import load_earth_relief
+from pygmt.helpers import GMTTempFile
+
+
+@pytest.fixture(scope="module", name="grid")
+def fixture_grid():
+    """
+    Load the grid data from the sample earth_relief file.
+    """
+    return load_earth_relief(
+        resolution="01d", region=[-5, 5, -5, 5], registration="pixel"
+    )
+
+
+def test_grdsample_file_out(grid):
+    """
+    grdsample with an outgrid set and the increment is changed.
+    """
+    with GMTTempFile(suffix=".nc") as tmpfile:
+        result = grdsample(grid=grid, outgrid=tmpfile.name, increment=[1, 2.5])
```

Comment:
Using `increment=[1, 2.5]` gives this warning. I think it's better to use a smaller y increment, like `[1, 0.5]`.
```
grdsample [WARNING]: Output sampling interval in y exceeds input interval and may lead to aliasing.
```

Reply:
```suggestion
        result = grdsample(grid=grid, outgrid=tmpfile.name, increment=[1, 0.5])
```